Tests/scripts/collect_tests/id_set.py

```diff
@@ -199,6 +200,8 @@ def __init__(self, marketplace: MarketplaceVersions) -> None:
 
             self.test_playbooks = self.id_to_test_playbook.values()
             self.modeling_rules = self.path_to_modeling_rule.values()
+            self.api_modules_to_integrations = {
+                script.object_id: script.imported_by for script in scripts if "ApiModule" in script.object_id}
```

Comment:
Consider changing both here and the other place with `object_id.endswith("ApiModule")` instead of `"ApiModule" in script.object_id`